recipes/nvidia-gpu/manifests/1000-state-driver.yaml

```
@@ -40,17 +40,18 @@ data:
     #!/bin/bash -x
 
     # Enable EUS and try to install kernel-devel and kernel-headers 
+    echo "Enable EUS and install kernel-devel kernel-headers  kernel-core rpms"
```

We know that this will fail in some OCP configurations, so maybe make it clear that it's only an alternative that we're trying:

```suggestion
    echo "Try to enable EUS and install kernel-devel kernel-headers kernel-core RPMs"
```

---

I would go for `Attempt to enable` , wdyt?

---

as you prefer, I personally prefer `Try to ...` but I'm not strong with any side